packages/animations/browser/src/render/shared.ts

```diff
@@ -151,12 +160,13 @@ if (_isNode || typeof Element !== 'undefined') {
   if (!isBrowser()) {
     _contains = (elm1, elm2) => elm1.contains(elm2);
   } else {
+    _documentElement = /* @__PURE__ */ (() => document.documentElement)();
```

Comment:
I don't like this and welcome suggestions on how to do this differently.

Reply:
Just curious why do we need an iffe here (is it due to the top-level assignment not being optimized by Terser)? If so, may be we can add a quick comment here for future context?

Reply:
There's a ban-toplevel-statement tslint rule (or a similar rule name, I don't recall by heart) that would fire if this is not wrapped inside an IIFE.

Reply:
Ok, got it. We do have some places where we disable that check via `// tslint:disable-next-line: no-toplevel-property-access`. However in this particular case I'm not sure if Terser would be able to tree-shake-away that code without that iffe.
```suggestion
    // tslint:disable-next-line: no-toplevel-property-access
    _documentElement = /* @__PURE__ */ document.documentElement;
```

Reply:
Yeah exactly. I also checked whether we have existing suppressions but the ones I can across were for const enums. In this case a bundler would likely assume a potential side-effect for the property read and be able to optimize less aggressively.

Reply:
Thanks for additional info. I think adding a quick comment (to explain why) here should be enough 👍